js/i18n.js

```diff
@@ -1,29 +1,36 @@
-/*
- * vim: ts=4:sw=4:expandtab
- */
-;(function() {
-    'use strict';
+/* eslint-env node */
 
-    // preload.js loads this, pulling it from main.js (where it was loaded from disk)
-    var messages = window.config.localeMessages;
-    var locale = window.config.locale;
+'use strict';
```

Comment:
Interesting, I didn’t realize this still was relevant for CommonJS but apparently it is. Apparently, JS modules have this implicitly so that’s another benefit of adopting `import` and `export` 😄 

P.S. I had to re-read specifics about this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode. Curious: What are your personal favorites about `use strict`?

Reply:
In the world of Node.js, I've have them at the top of all files for so long I don't even think about it any more. And I turn on the eslint rules to make it so I don't forget.

In this case I think I ran into it because the `js/` dir doesn't have an eslint config that specifies that these files are modules. If we're not getting the benefits of strict mode now, then we should certainly do what it takes to get it!